deployment/terraform/modules/aws/vpc/outputs.tf

```diff
@@ -13,3 +13,8 @@ output "public_subnets" {
 output "vpc_cidr_block" {
   value = module.vpc.vpc_cidr_block
 }
+
+output "s3_vpc_endpoint_id" {
+  description = "ID of the S3 gateway VPC endpoint created for this VPC"
+  value       = aws_vpc_endpoint.s3.id
```

Comment:
Should this error if defined _and_ create_vpc is true? Otherwise, this would get silently ignored which seems like a potential pitfall